scripts/src/admin/services.ts

```diff
@@ -693,18 +693,23 @@ export async function changeOffer(body: ChangeOfferData, params: { offer_id: str
 	await offer.save();
 }
 
-type UpdateAppConfigRequest = Request & {
-	body: ApplicationConfig
-};
+type AppConfig = ApplicationConfig & { [limits: string]: number };
+type UpdateAppConfigRequest = Request & { body: AppConfig; };
 
 export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
+	const config: AppConfig = req.body;
```

Comment:
Nice. Glad to see there's someone else not too lazy to do this.

Reply:
@liorama 
it's not about being lazy or not, i think it's redundant.

also, here, there's no need for the type annotation.